web/settings.py

```diff
@@ -31,6 +31,15 @@
 EVENT_TICKET_EMAIL = "ticket@makentnu.no"
 EMAIL_SITE_URL = "https://makentnu.no"
 
+
+SENDGRID_API_KEY = os.getenv('SENDGRID_API_KEY')
+
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'test.makentnu@gmail.com'
+EMAIL_HOST_PASSWORD = 'makentnu'
```

Comment:
Never commit passwords or similar secrets, especially when this is a public repository. Please replace this line with something like
```
def get_email_password():
    email_secret_file = os.path.join(BASE_DIR, "email_secret.txt")
    try:
        with open(email_secret_file) as f:
            return f.read().strip()
    except IOError:
        if not DEBUG:
            raise FileNotFoundError(f"Please create the file \"{email_secret_file}\" with the password to {EMAIL_HOST_USER}")


EMAIL_HOST_PASSWORD = get_email_password()
```
and make sure to create the file `email_secret.txt` in the `BASE_DIR` of the server containing the password to test.makentnu@gmail.com.

Reply:
This should be done in the `local_settings.py` file instead, it's not tracked by git and used in addition to the main settings file. The server already has these variables set so it sends from `noreply@makentnu.no`

Reply:
So remove these lines, move them to `web/local_settings.py` locally, and change the password of the email account.

Reply:
The server may actually send from any `@makentnu.no` email address (even if the email address is not manually created). So this should probably be set to send from `sla-request@makentnu.no` or something similar.

Reply:
@zootos But this is the email used in the settings file, which shouldn't be changed from the default. If possible each different application should use a different appropriate address.

Reply:
The `noreply@makentnu.no` defined for `EMAIL_HOST_USER` just defines the which GSuite user to use. I have whitelisted the IP of the server with the `noreply@makentnu.no` user in GSuite to be allowed to send emails from whatever `@makentnu.no` email trough the GSuite SMTP relay. So with the current configuration on the server, we can set the `from` field to be any `@makentnu.no` email.